app/src/cose.h

```diff
@@ -91,6 +196,36 @@ namespace scitt::cose
       }
       return false;
     }
+
+    // Returns a bool representing whether the input label is a known
+    // parameter in the context of a notary profile.
+    bool is_known_in_notary(
```

Comment:
I don't think profile-specific code (apart from decoding) should be in `cose.h`. Could this move into `verifier.h`?